src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -1492,3 +1492,25 @@ function prefix(what, func, fgc, bgc, window)
   func(window,what)
   resetFormat(window)
 end
+
+--- Moves the cursor in the given window up a specified number of lines
+--- @param windowName Optional name of the window to use the function on
+--- @param lines number of lines to move cursor
+function moveCursorUp(windowName, lines)
+  if type(window) ~= "string" then  lines, window = window, "main" end
+  lines = tonumber(lines) or 1
+  local curLine = getLineNumber(window)
+  if not curLine then error("moveCursorUp: window does not exist", 2) end
+  moveCursor(window, 0, math.max(curLine - lines, 0))
```

Comment:
This will move the cursor to the beginning of the given line. Reading the comment, I'd expect it to move to the current column.

Reply:
Hm... I think moving the cursor to the start of the line would be more intuitive behaviour (as it is now).

Reply:
That makes sense to me and is how I imagined it as well.

Reply:
Looks like we identified two perfectly valid usecases here - let's make it work for both? @JorMox